packages/starrating/vars.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/vars')
```

Comment:
I don't think there are any vars in this package.  I am looking through it wondering if there should be, but I'm not coming up with the traditional vars that are usually shared between react and css.  For instance, APPEARANCES is only used in react/star.js.  Do you know of any?

Reply:
no there are no vars to export

Reply:
if we add sizes(probably reexport icon sizes) there will be but right now there is only the one size